tools/getrealdeps.rb

```diff
@@ -1,30 +1,39 @@
 #!/usr/bin/env ruby
-# getrealdeps for Chromebrew version 1.2
+# getrealdeps for Chromebrew version 1.3
 # Author: Satadru Pramanik (satmandu) satadru at gmail dot com
 require 'fileutils'
 
+if ARGV.include?('--use-crew-dest-dir')
+  ARGV.delete('--use-crew-dest-dir')
```

Comment:
Why does the argument need to be deleted?

Reply:
plumbing for allowing getrealdeps to handle multiple package arguments. I just didn't implement a for loop to do that.

Reply:
Why not?  That would be the elegant way to do it, right?

Reply:
Because that will require refactoring to wrap everything in a function, which I haven't done yet.

But also, removing the argument allows argv[0] to be the package that is looked at.

Reply:
Wrapped!

Reply:
For a future PR then?